pointcept/engines/hooks/evaluator.py

```diff
@@ -150,18 +152,23 @@ def eval(self):
             self.trainer.storage.put_scalar("val_intersection", intersection)
             self.trainer.storage.put_scalar("val_union", union)
             self.trainer.storage.put_scalar("val_target", target)
-            self.trainer.storage.put_scalar("val_loss", loss.item())
+            self.trainer.storage.put_scalar(
+                "val_loss", loss.item() if loss is not None else 0
```

Comment:
Handles the case when the step is skipped and the loss is None.

Reply:
Hey @Streakfull,

I think if data doesn't contain a valid label, maybe we should filter is out during the data preprocessing. Maybe using a JSON file to record valid data is also a good way. Skipping the training step might also cause some problems like the scheduler is not updated.